test/e2e/framework.go

```diff
@@ -125,7 +125,7 @@ const (
 	flowAggregatorConfName   = "flow-aggregator.conf"
 
 	agnhostImage        = "registry.k8s.io/e2e-test-images/agnhost:2.40"
-	ToolboxImage        = "antrea/toolbox:1.5-1"
+	ToolboxImage        = "antrea/toolbox:latest"
```

Comment:
Why to change this to the latest image?

Reply:
I think we should avoid using the latest version because it may cause unexpected CI failures when the latest release is updated.

Reply:
Done.